org.jacoco.core.test.validation.java14/pom.xml

```diff
@@ -35,4 +35,25 @@
       <version>${project.version}</version>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <compilerArgs>
+            <arg>--enable-preview</arg>
+          </compilerArgs>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>--enable-preview</argLine>
```

Comment:
Out of curiosity: Why does surefire enable this flag?

Reply:
@marchof enablement of Preview Features by `--enable-preview` flag is required both at compile and at run time - see [Preview Language and VM Features (JEP 12)](https://openjdk.java.net/jeps/12), otherwise we'll get

```
java.lang.UnsupportedClassVersionError: Preview features are not enabled for org/jacoco/core/test/validation/java14/RecordsTest (class file version 58.65535). Try running with '--enable-preview'
```

Reply:
Right, thanks!